js_modules/dagster-ui/packages/ui-core/src/ui/Filters/useFilter.tsx

```diff
@@ -72,9 +72,15 @@ export const FilterTagHighlightedText = React.forwardRef(
   ) => {
     return (
       <FilterTagHighlightedTextSpan
-        text={children}
+        text={
+          <>
+            {children}
+            <div style={{display: 'none'}}>…</div>
```

Comment:
Why `display: none` on this?

Reply:
oh wait jk, I need this to trick to make the tooltip text show up on hover even though the text isnt truncated. It's a bit of a hack.